spec/integration/add_transactions_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.